far/FarEng.hlf.m4

```diff
@@ -5372,25 +5370,29 @@ If current value of an option is other than the default, the option is marked wi
 
 @Codepages.NoAutoDetectCP
 $ #far:config Codepages.NoAutoDetectCP#
-    Это строка, при помощи которой можно задать кодовые страницы, которые будут исключены
-из автоопределения кодовой страницы UCD (Universal Codepage Detector). Иногда (особенно на
-небольших файлах) UCD назойливо выбирает неподходящие кодовые страницы.
-
-    Умолчательное значение это пустая строка #""#, в этом случае все кодовые страницы которые
-может выдать UCD (около двух десятков - гораздо меньше чем может быть установлено на компьютере)
-разрешены.
-
-    Если параметр равен строке #"-1"#, то в зависимости от того включён или нет показ
-только избранных кодовых страниц (#Ctrl+H# в ~меню кодовых страниц~@CodePagesMenu@),
-будут разрешены только стандартные (ANSI, OEM, Unicode) и избранные, либо все.
-
-    В противном случае параметр должен быть списком запрещённых номеров кодовых страниц.
-Например "1250,1252,1253,1255,855,10005,28592,28595,28597,28598,38598".
-
-    Юникодные кодовые страницы (1200, 1201, 65001) проверяются отдельно от UCD, поэтому не могут быть
-отключены, даже если они есть в списке исключения.
+    This string parameter defines the code pages, which will be excluded
```

Comment:
This is a restrictive clause, no comma is required.

Reply:
Thank you. My bad.